src/styles/main.scss

```
@@ -0,0 +1,42 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
+  padding-inline: $cellPading;
+
+  &__day {
+    width: $cellSize;
+    height: $cellSize;
+    background-color: $cellCollor;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    border: solid 1px black;
+
+    font-family: $cellFontFamily;
+    font-size: $cellFontHeight;
```

For a smooth movement of the day to the place
```suggestion
    font-size: $cellFontHeight;
    transition: transform $transformDuration;
```